src/baldr/edgeinfo.cc

```diff
@@ -82,6 +90,13 @@ EdgeInfo::EdgeInfo(char* ptr, const char* names_list, const size_t names_list_le
     extended_wayid3_ = static_cast<uint8_t>(*ptr);
     ptr += sizeof(uint8_t);
   }
+  if (ei_.had_conditional_speed_limits) {
+    conditional_speed_limits_count_ = static_cast<uint8_t>(*ptr);
+    ptr += sizeof(uint8_t);
+
+    conditional_speed_limits_ = reinterpret_cast<ConditionalSpeedLimit*>(ptr);
+    ptr += conditional_speed_limits_count_ * sizeof(ConditionalSpeedLimit);
+  }
 
   // Set encoded elevation pointer
   encoded_elevation_ = reinterpret_cast<int8_t*>(ptr);
```

Comment:
Unfortunately there was not enough space to put count into the header, just a single bit, so the size goes here.
I put `conditional_speed_limits_` **before** elevation as its size is unknown in the `EdgeInfo` constructor and it was quite hard to pass it everywhere.